.github/ISSUE_TEMPLATE/end_game.md

```diff
@@ -46,6 +46,7 @@ assignees: ''
 - [ ] Discord announcement to [#announcements](https://discord.gg/zv8ahzZVpf)
 - [ ] For major releases, [blog.spiceai.org](https://blog.spiceai.org) announcement
 - [ ] Update `version.txt` and version in `Cargo.toml` to the next release version.
+- [ ] Update versions in [brew taps](https://github.com/spiceai/homebrew-spiceai)
```

Comment:
Is there a doc or a GH action (in homebrew-spiceai) to do this?

Reply:
No GH action yet. but will be nice to have one soon.

The tap is pretty self explanatory, let's merge this for now and add the doc/automation if it is needed.